### PR TITLE
WIP: Implemented workaround for GTK assertions when starting Tribler on Ubuntu or Debian

### PR DESCRIPTION
**Pull request by devos50**
Placing a `wx.SearchCtrl` widget inside a container makes GTK happy.

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis can you review?

File: Tribler/Main/vwxGUI/list_header.py [hunk 1]
```diff
@@ -629,12 +629,15 @@ def toggle_grid(event):
 
         self.grid_button.Bind(wx.EVT_LEFT_UP, toggle_grid)
 
+        # Martijn: Workaround for assertions on Linux when adding a wx.SearchCtrl widget to a wx.Sizer
+        search_container_panel = wx.Panel(panel)
+
         if sys.platform == 'darwin':
-            self.search = wx.TextCtrl(panel)
+            self.search = wx.TextCtrl(search_container_panel)
```
Comment (whirm) [inline Tribler/Main/vwxGUI/list_header.py]: Except for the size, the rest could be moved out of the if/else, right?

Comment (lfdversluis) [inline Tribler/Main/vwxGUI/list_header.py]: I don think so. `SetHint` vs `SetDescriptiveText` is probably the reason.

**Comment by lfdversluis** [comment]
Workaround for wx are ok. I do not know if we want to keep the suggestions, but if not it's fine with me!

File: Tribler/Main/vwxGUI/list_header.py (hunk 1 above)
Comment (devos50) [inline Tribler/Main/vwxGUI/list_header.py]: @whirm nope because of the reason @lfdversluis gave.

**Comment by devos50** [comment]
@lfdversluis well I also don't mind if we drop the suggestions, however, this seems to work fine (see screenshots made during the tests).

File: Tribler/Main/vwxGUI/list_header.py (hunk 1 above)
Comment (whirm) [inline Tribler/Main/vwxGUI/list_header.py]: oh, sorry somehow I thought you where undoing the OSX hack because SearchCtrl worked fine now.

Comment (whirm) [inline Tribler/Main/vwxGUI/list_header.py]: Have you tried if that makes OSX happy BTW?

Comment (devos50) [inline Tribler/Main/vwxGUI/list_header.py]: Yes, works fine!

Comment (devos50) [inline Tribler/Main/vwxGUI/list_header.py]: @whirm it also works on Windows

Comment (whirm) [inline Tribler/Main/vwxGUI/list_header.py]: no, I mean, if going back to using SearchCtrl with the fixes in this PR works

Comment (devos50) [inline Tribler/Main/vwxGUI/list_header.py]: No that widget is still bugged :/

**Comment by devos50** [comment]
@whirm I accidentally forgot to push a minor change but for now I'm done with this PR.

**Comment by devos50** [comment]
I already start liking our clean test log files!

**Pull request merged**